hack/cmds/commands.json

```diff
@@ -6040,6 +6040,16 @@
         "type": "command",
         "optional": true
       },
+      {
+        "command": "DELREF",
+        "type": "command",
+        "optional": true
+      },
+      {
+        "command": "ACKED",
+        "type": "command",
+        "optional": true
+      },
```

Comment:
Hi @Ryan2327, they should be in one enum type. Please take a look at https://github.com/redis/rueidis/pull/882/files. Also, they should be put behind the LIMIT option.